os/arch/arm/src/common/up_createstack.c

```diff
@@ -106,6 +110,12 @@
 #define STACK_ALIGN_DOWN(a) ((a) & ~STACK_ALIGN_MASK)
 #define STACK_ALIGN_UP(a)   (((a) + STACK_ALIGN_MASK) & ~STACK_ALIGN_MASK)
 
+#ifdef CONFIG_MPU_STACKGUARD
+#define STACK_GUARD_ADDR_MASK	0xffffffe0	/* Bits N-31:  Used for MPU Region base addrese */
+extern uint32_t g_mpu_region_nr;
+void mpu_configure_app_regs(uint32_t *regs, uint32_t region, uintptr_t base, size_t size, uint8_t readonly, uint8_t execute);
```

Comment:
Let's move the definition of prototype in header and let's just include the header.

Reply:
done